libcontainer/nsenter/nsexec.c

```diff
@@ -12,7 +13,6 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/ioctl.h>
-#include <sys/socket.h>
```

Comment:
@hqhq It appears older versions required sys/socket.h to be included before including linux/netlink.h:
http://comments.gmane.org/gmane.linux.network/203185